### PR TITLE
[stable/selenium] upgrade version to 3.14.0

### PR DESCRIPTION
**Pull request by Chr1st0ph**
upgrade version to 3.14.0

### CODE REVIEW

File: stable/selenium/values.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ hub:
 
   ## The tag for the image
   ## ref: https://hub.docker.com/r/selenium/hub/tags/
-  tag: "3.13.0"
+  tag: "3.14.0"
```
Comment (davidkarlsen) [inline stable/selenium/values.yaml:15]: can you update the readme to the according values (for all of them)

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/selenium/values.yaml (hunk 1 above)
Comment (Chr1st0ph) [reply stable/selenium/values.yaml:15]: @davidkarlsen done

**Comment by mattfarina** [comment]
@flah00 @diemol Can you please review this PR?

**Comment by cpanato** [comment]
I can help if need. I tested locally and look good so far

**Pull request merged**